install/operator/pkg/syndesis/capabilities/capabilities.go

```diff
@@ -84,6 +87,17 @@ func ApiCapabilities(clientTools *clienttools.ClientTools) (*ApiServerSpec, erro
 
 	apiSpec.Version = info.Major + "." + info.Minor
 
+	// in openshift 3, the minor may contains a "+" suffix, so we should remove it
+	minor, err := strconv.Atoi(strings.ReplaceAll(info.Minor, "+", ""))
+    if err != nil {
+        fmt.Printf("Trying to determine if running on openshift 4, the conversion from string to integer failed for minor version %s. %s\n", info.Minor, err)
+    }
+	if minor >= 13 {
+		apiSpec.Openshift4 = true
```

Comment:
Can we not detect whether its openshift 4 through the existence of certain CRD version rather than specifying an "Are you an openshift 4" test?

Reply:
Sure, I will change to detect a CRD instead.